### PR TITLE
Add fixed markdown documentation

### PR DESCRIPTION
**Pull request by m-niesluchow**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -62,13 +62,23 @@ TAU Profiles (mobile, TV, wearable) have differences, so some of the examples wi
 ## Where to get it
 
 The framework is included on all Tizen devices and the SDK emulator (starting from Tizen 2.3). Just create
-a new web application in the Tizen SDK IDE.
+a new web application in the Tizen SDK IDE. See also our [github repository](https://github.com/Samsung/TAU).
 
 ## What next?
```
Comment (lslachciak) [inline docs/index.md]: Please add link to TAU API documentation too
https://developer.tizen.org/development/api-references/web-application

**Comment by m-niesluchow** [comment]
@lslachciak 
Revert removed

File: docs/index.md (hunk 1 above)
Comment (m-niesluchow) [reply docs/index.md]: Done

File: docs/index.md [hunk 2]
```diff
@@ -62,13 +62,23 @@ TAU Profiles (mobile, TV, wearable) have differences, so some of the examples wi
 ## Where to get it
 
 The framework is included on all Tizen devices and the SDK emulator (starting from Tizen 2.3). Just create
-a new web application in the Tizen SDK IDE.
+a new web application in the Tizen SDK IDE. See also our [documentation](https://developer.tizen.org/dev-guide/5.0.0/org.tizen.web.apireference/html/ui_fw_api/ui_fw_api_cover.htm) and [github repository](https://github.com/Samsung/TAU).
```
Comment (lslachciak) [inline docs/index.md]: documentation --> API documentation
why link is more specific not just https://developer.tizen.org/development/api-references/web-application

**Pull request merged**